install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -11,144 +11,396 @@
+(unless the project has been customized for access to this NFS export). So, in this scenario, the above
+pod will fail because its group id of *5555* is not within the SCC's or the namespace's range of allowed
+group ids. 
+
+[[scc-supplemental-groups]]
+==== Supplemental Groups and Custom SCCs
+To remedy this situation a custom SCC can be created such that a min and max group id are defined,
+id range checking is enforced, and the group id of 5555 is allowed. It is considered a better
+practice to create new SCCs versus modifying a predefined SCC, or changing the range of allowed
+ids in the predefined projects. 
+
+The easiest way to create a new SCC is to export an existing SCC and customize the yaml file to 
+meet the requirements of the new SCC. For example:
+```
+# oc export SCC restricted >new-scc.yaml <1>
+##edit new-scc.yaml file
+# oc create -f new-scc.yaml <2>
+```
+<1> use the _restricted_ SCC as a template for the new SCC.
+<2> instantiate the new SCC
+
+*Note:* the `oc edit scc` command can be used to modify an instantiated SCC.
+
+Here is a fragment of a new SCC named "nfs-scc":
+```
+# oc export scc nfs-scc 
+allowHostDirVolumePlugin: false  #the allow* bools are the same as for the "restricted" scc
+...
+kind: SecurityContextConstraints
 metadata:
-  name: ebs-web
+  ...
+  name: nfs-scc <1>
+priority: 9 <2>
+...
+supplementalGroups:
+  type: MustRunAs <3>
+  ranges:
+  -  min: 5000 <4>
+     max: 6000
+...
+```
+<1> the name of the new SCC.
+<2> numerically larger numbers have greater priority, nil or omitted is the lowest priority.
+Higher priority SCCs sort before lower pri SCCs and thus have a better chance of matching a new pod
+<3> `supplementalGroups` is a strategy and it is set to _MustRunAs_, which means group id checking
+is required.
+<4> multiple ranges are supported. The allowed group id range here is 5000-5999, with the default
+supplemental group being 5000.
+
+When the same pod shown above runs against this new SCC (assuming, of course, the pod has access
+to the new SCC), it will start because the group *5555*, supplied in the pod spec, is now allowed
+by the custom SCC.
+
+[[fsgroup]]
+== FS Group
+*Note:* the link:#scc[SCC overview], above, should be read before working with FS groups.
+
+`*fsGroup*` defines a pod's "file system group" id, which gets added to the container's supplemental
+groups. As mentioned link:#supplemental-groups[above], the `supplementalGroups` id applies to shared
+storage; whereas, the `fsGroup` id is used for block storage.
+
+Block storage, such as Ceph-RBD, iSCSI, and various cloud storage, is typically dedicated to a single
+pod which has requested the block storage volume, either directly or via a persistent volume claim (PVC).
+Unlike shared storage, block storage is *_taken over_* by a pod, meaning that user and group ids supplied
+in the pod spec (or image) are applied to the actual, physical block device. Typically, block storage is
```

Comment:
Here's the explanation: My ose env has a single node so both ceph pods are scheduled on the same host. In this case the ceph PV can be shared (regardless of the readonly setting). However, there's a k8s PR (https://github.com/kubernetes/kubernetes/pull/15956) to make the scheduler sensitive to rbd noDiskConflicts. Base on this forward direction I will remove the sentence that mentions the ability to share block storage.